scripts/setup-sources.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# A quick and dirty script to prepare the source code needed for
+# the apalache bench tests
+
+set -euf -o pipefail
+
+latest="0.7.0"
+devl_dir="${HOME}/devl"
+
+mkdir -p "$devl_dir"
+git clone git@github.com:informalsystems/apalache.git "$devl_dir/apalache" > /dev/null
+
+for v in "0.5.1" "0.6.0" "0.7.0"
```

Comment:
Note that there is no tagged release for version 0.5.2, which is the one version specififed in the makefile. I need a pointer here about how we reconcile that.

Reply:
I don't know how we missed that. Just added `v0.5.2` for the commit that we used to build 0.5.2. It is a backport of the build system from 0.6.0 to 0.5.1.

Reply:
Ah! That makes sense. I tried to run 0.5.1, only to see it fail because there was no makefile :D